### DISCUSSION TITLE
(WARNING) Main token does not have a billing source, some codes will not be sniped.

### DISCUSSION THREAD

**Discussion by Dominikhun250** [GiorgioBrux/nitro-sniper-enhanced#209] [Q&A] [answered]
How i need understand this?
(WARNING) Main token does not have a billing source, some codes will not be sniped.

**Comment by GiorgioBrux** [comment, marked as answer]
Hey, I need to double check this info, but at least as of a few years ago -- if you don't add a payment method to your main account then you won't be able to redeem some types of code. 

Will double check if this is still the case and remove the warning if it's outdated.
